contracts/api-spec.yaml

```diff
@@ -409,8 +458,8 @@ components:
           type: string
           example: 'v7BM3ejS'
 
-    OutputChannelModel:
-      description: A channel through which an encoder outputs a video stream
+    ChannelModel:
+      description: Parent for input and output channels
```

Comment:
I'm kinda iffy on that description but that might just be a consequence of the design being a bit strange